### PR TITLE
add keyword to control the z-axis of vol-depol colorplot

### PR DESCRIPTION
**Pull request by HolgerPollyNet**
I updated only the 2 files pollyxt_display_rcs.m and .py to allow for adjusting the z-scale for the volume depol time-height plots. Please forget about the other files.....globa config was also adjusted with respect to that....

### CODE REVIEW

File: lib/polly_general_func_lib/pollyxt_display_rcs.m [hunk 1]
```diff
@@ -30,6 +30,8 @@ function pollyxt_display_rcs(data, taskInfo, config)
 RCS1064FRColorRange = config.zLim_FR_RCS_1064;
 RCS355NRColorRange = config.zLim_NR_RCS_355;
 RCS532NRColorRange = config.zLim_NR_RCS_355;
+Voldepol355ColorRange = config.zLim_VolDepol_355;
+Voldepol532ColorRange = config.zLim_VolDepol_532;
```
Comment (ZPYin) [inline lib/polly_general_func_lib/pollyxt_display_rcs.m:34]: Here, you used different parameters than what you configured in the `polly_global_config.json` (zLim_NR_VDP_355 and zLim_NR_VDP_532).

File: lib/polly_general_func_lib/pollyxt_display_OC_att_beta.py [hunk 2]
```diff
@@ -183,8 +183,8 @@ def pollyxt_display_OC_att_beta(tmpFile, saveFolder):
         )
 
     ax.set_title(
-        'Merged Attenuated Backscatter at {wave}nm'.format(wave=355) +
-        ' from {instrument} at {location}'.format(
+        'Merged Attenuated Backscatter Coefficient at {wave}nm'.format(wave=355) +
```
Comment (ZPYin) [inline lib/polly_general_func_lib/pollyxt_display_OC_att_beta.py:186]: Such a long string for figure title could probably be cropped by the figure frame. But let's see what it looks like after being merged. Later on, I will think about adding new lines if it is too long.

File: lib/polly_general_func_lib/pollyxt_display_rcs.py [hunk 3]
```diff
@@ -415,7 +415,9 @@ def pollyxt_display_rcs(tmpFile, saveFolder):
     ax = fig.add_axes([0.11, 0.15, 0.79, 0.75])
     pcmesh = ax.pcolormesh(
         Time, Height, volDepol_532,
-        vmin=0.0, vmax=0.3, cmap=cmap, rasterized=True)
+        vmin=Voldepol532ColorRange[0],
```
Comment (ZPYin) [inline lib/polly_general_func_lib/pollyxt_display_rcs.py:418]: The variables of `Voldepol532ColorRange` and `VolDepol355ColorRange` need to be loaded from MATLAB .mat files. See https://github.com/PollyNET/Pollynet_Processing_Chain/blob/c82f1e5db3cca317161bda5265690b97d035eb21/lib/polly_general_func_lib/pollyxt_display_rcs.py#L126-L129

**Comment by ZPYin** [comment]
Please try to name the PRs properly, in case it can be easily searched in future.

i.e., `add keyword to control the z-axis of vol-depol colorplot` for this PR

File: lib/polly_general_func_lib/pollyxt_display_rcs.m (hunk 1 above)
Comment (HolgerPollyNet) [reply lib/polly_general_func_lib/pollyxt_display_rcs.m:34]: fixed

File: lib/polly_general_func_lib/pollyxt_display_OC_att_beta.py (hunk 2 above)
Comment (HolgerPollyNet) [reply lib/polly_general_func_lib/pollyxt_display_OC_att_beta.py:186]: ok, but we can also skip, as you like

File: lib/polly_general_func_lib/pollyxt_display_rcs.py (hunk 3 above)
Comment (HolgerPollyNet) [reply lib/polly_general_func_lib/pollyxt_display_rcs.py:418]: fixed

**Comment by HolgerPollyNet** [comment]
I pushed the new commits to Holger, how can I conclude them in this pull request?

**Comment by ZPYin** [comment]
> I pushed the new commits to Holger, how can I conclude them in this pull request?

It has been updated automatically. I will merge it and then start testing phase.

File: lib/polly_general_func_lib/pollyxt_display_rcs.py (hunk 3 above)
Comment (ZPYin) [reply lib/polly_general_func_lib/pollyxt_display_rcs.py:398]: seems you didn't fix this yet. But this is a very minor issue, I will do it after the PR was merged.

Remember to pull it back to your local `dev` branch.

**Comment by ZPYin** [comment]
BTW, another tip: please pull the latest `dev` branch and merge it locally before you send the PRs. 

By this way, it can make sure you resolved all conflicts in the first place. Otherwise, it will cost extra time for code viewers to think about what should be remained.

**Comment by HolgerPollyNet** [comment]
Thanks you are completely right! I will do this. I also used git gui
    this time and obviously it was not working so well like the command
    line approach.....
    
    Is the option already implemented at the server? Or do you have
    developments ongoing prohibiting this?
    
    Thanks
    Holger
    
    Am 10.06.2020 um 02:04 schrieb ZPYin:
    
    
      
      BTW, another tip: please pull the latest dev
        branch and merge it locally before you send the PRs.
      By this way, it can make sure you resolved all conflicts in the
        first place. Otherwise, it will cost extra time for code viewers
        to think about what should be remained.
      —
        You are receiving this because you authored the thread.
        Reply to this email directly, view it on GitHub, or unsubscribe.
      [
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/PollyNET/Pollynet_Processing_Chain/pull/62#issuecomment-641642511",
"url": "https://github.com/PollyNET/Pollynet_Processing_Chain/pull/62#issuecomment-641642511",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
    
    
    -- 
*****************************************************
 Dr. Holger Baars (Dipl.-Met.)
 Leibniz Institute for Tropospheric Research (TROPOS)
 Permoserstr. 15
 04318 Leipzig, Germany
 phone: ++49 341 2717 7314
 fax: ++49 341 2717 99 7314
 email: baars@tropos.de
 internet: http://www.tropos.de
 ****************************************************

**Comment by ZPYin** [comment]
>  Is the option already implemented at the server? Or do you have developments ongoing prohibiting this?

It has been implemented in the server, but at the `dev` branch only. Make sure you are working at the `dev` branch if you want to use this setting.

**Pull request merged**